src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.js

```diff
@@ -0,0 +1,51 @@
+import path from 'ramda/src/path';
+import pathToRegexp from 'path-to-regexp';
+import {
+  getVariant,
+  variantSanitiser,
+  variants,
+} from '#lib/utilities/variantHandler';
+import { getPreferredVariant } from '#app/contexts/UserContext/cookies';
+
+const getVariantRedirectUrl = (props, service, variant) => {
+  const cookieVariant = getPreferredVariant(service);
+  const defaultVariant = getVariant({
+    service,
+    variant: variantSanitiser(variant),
+  });
+
+  if (!props) return null;
+
+  const match = path(['match'], props, {});
```

Comment:
does `path` even accept a 3rd argument?

Reply:
I must have been thinking of `pathOr(...)` ... Will fix. Thanks!

Reply:
Fixed!